client/my-sites/site-settings/manage-connection/site-ownership.jsx

```diff
@@ -35,16 +39,53 @@ class SiteOwnership extends Component {
 		);
 	}
 
-	renderCurrentUser() {
+	isUserExcludedFromSelector = user => {
 		const { currentUser } = this.props;
+		return (
+			user.linked_user_ID === false ||
+			user.linked_user_ID === currentUser.ID ||
+			! includes( user.roles, 'administrator' )
+		);
+	};
+
+	transformUser( user ) {
+		return { ...user.linked_user_info, ...{ ID: user.ID } };
+	}
+
+	onSelect = user => {
+		const { translate } = this.props;
+
+		accept(
+			translate( 'Are you sure you want to transfer ownership?' ),
+			accepted => {
+				accepted && this.props.changeOwner( this.props.siteId, user.ID, user.name );
+			},
```

Comment:
fun fact!

there's no need for the `{` and `}` here. as an arrow function it will evaluate the expression and return its value, which we ignore. in cases like this I like to do that since it clears up some noise. it's a personal preference though so feel free to ignore.